kuzzleio/sdk-cpp
# Rename enums

## What does this PR do ?

Rename enums because of https://github.com/kuzzleio/sdk-c/pull/45

### How should this be manually tested?

check travis

https://github.com/kuzzleio/sdk-c/pull/45 :arrow_left: 
:arrow_right: https://github.com/kuzzleio/documentation-V2/pull/179
